src/cleanup-java.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from '@actions/core';
+import * as gpg from './gpg';
+
+async function run() {
+  if (core.getInput('gpg-private-key', {required: false})) {
+    console.log('removing private key from keychain');
+    try {
+      const keyFingerprint = core.getState('gpg-private-key-fingerprint');
```

Comment:
There are a bunch of constants floating around such as `gpg-private-key-fingerprint` and `gpg-private-key`. Could we throw those into a single file such as `constants.ts`?